README.md

```diff
@@ -1,172 +1,151 @@
 
-## Introduction
+<p align="center">
+<a href="https://opensource.org/licenses/Apache-2.0"><img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
+<a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/License-MIT-blue.svg" /></a>
+<a href=""><img src="https://img.shields.io/badge/nim-%3E%3D1.0.6-orange.svg?style=flat-square" /></a>
+<a href=""><img src="https://img.shields.io/badge/go-%3E%3D1.12-orange.svg?style=flat-square" /></a>
+</p>
 
-### Running against the Go daemon
+## Introduction
 
-An implementation of [libp2p](https://libp2p.io/) in Nim, as a wrapper of the [Libp2p Go daemon](https://github.com/libp2p/go-libp2p).
+An implementation of [libp2p](https://libp2p.io/) in Nim. Also provides a Nim wrapper of the [Libp2p Go daemon](https://github.com/libp2p/go-libp2p).
 
-Note that you need Go 1.12+ for the below instructions to work!
+## Project Status
+The current native Nim libp2p implementation support is experimental and shouldn't be relied on for production use. It is however under active development and we hope to achieve a reasonable level of stability in the upcoming months, as we will be integrating it across our own set of products, such as the Nim Beacon Chain. <br>
 
-Install dependencies and run tests with:
+Check our [examples folder](/examples) to get started! 
 
-```bash
-git clone https://github.com/status-im/nim-libp2p && cd nim-libp2p
-nimble install
-nimble test
-git submodule update --init --recursive
-go version
-git clone https://github.com/libp2p/go-libp2p-daemon
-cd go-libp2p-daemon
-git checkout v0.0.1
-go install ./...
-cd ..
-```
+# Table of Contents
+- [Background](#background)
+- [Install](#install)
+  - [Prerequisite](#prerequisite)
+- [Usage](#usage)
+  - [API](#api)
+  - [Getting Started](#getting-started)
+  - [Tutorials and Examples](#tutorials-and-examples)
+  - [Using the Go Daemon](#using-the-go-daemon)
+- [Development](#development)
+  - [Tests](#tests)
+  - [Packages](#packages)
+- [Contribute](#contribute)
+  - [Core Developers](#core-developers) 
+- [License](#license)
 
-Try out the chat example:
+## Background   
+libp2p is a networking stack and library modularized out of [The IPFS Project](https://github.com/ipfs/ipfs), and bundled separately for other tools to use.
 
-```bash
-nim c -r --threads:on examples\chat.nim
-```
+libp2p is the product of a long, and arduous quest of understanding -- a deep dive into the internet's network stack, and plentiful peer-to-peer protocols from the past. Building large scale peer-to-peer systems has been complex and difficult in the last 15 years, and libp2p is a way to fix that. It is a "network stack" -- a protocol suite -- that cleanly separates concerns, and enables sophisticated applications to only use the protocols they absolutely need, without giving up interoperability and upgradeability. libp2p grew out of IPFS, but it is built so that lots of people can use it, for lots of different projects.
```

Comment:
```suggestion
libp2p is the product of a long and arduous quest of understanding; a deep dive into the internet's network stack and the peer-to-peer protocols from the past. Building large scale peer-to-peer systems has been complex and difficult in the last 15 years and libp2p is a way to fix that. It is a "network stack", a suite of networking protocols that cleanly separates concerns and enables sophisticated applications to only use the protocols they absolutely need, without giving up interoperability and upgradeability. 

libp2p grew out of IPFS, but it is built so that lots of people can use it, for lots of different projects.
```